mgmt/rest/v1/task.go

```diff
@@ -45,11 +45,12 @@ var (
 )
 
 func (s *apiV1) addTask(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
-	task, err := core.CreateTaskFromContent(r.Body, nil, s.taskManager.CreateTask)
-	if err != nil {
-		rbody.Write(500, rbody.FromError(err), w)
+	task, errs := core.CreateTaskFromContent(r.Body, nil, s.taskManager.CreateTask)
```

Comment:
`core.CreateTaskFromContent` returns `serr`, not slice of errors. Name of variable `errs` is misleading! What is more, checking `len(errs.Error()) != 0 ` does not have sense (see the [line 49](https://github.com/intelsdi-x/snap/pull/1701/files#diff-f7e0c90ee57f3f6bd2ab3a6199986599R49)